<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscountOffer</class>
 <widget class="QDialog" name="DiscountOffer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>661</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>569</width>
    <height>661</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>569</width>
    <height>661</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Discount Offer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: rgb(52, 52, 52);
}
QPushButton
{
	
	color: rgb(255, 255, 255);
	font-size: 17px;
	border: 2px solid rgb(255, 255, 255);
	background-color: rgb(52, 52, 52);
}
QPushButton:hover
{
	color: rgb(52, 52, 52) ;
	background-color: rgb(255, 255, 255);
}
QLabel
{
	font-size: 20px;
	color: rgb(191, 191, 191);
}
QLineEdit
{	
	color: rgb(255, 255, 255);
	border-style: solid;
	background-color: rgb(30, 30, 30);
	font-size: 40px;
}
QComboBox
{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	font-size: 17px;
}
QComboBox:hover
{
	background-color: rgb(0, 0, 0);
}
QComboBox QAbstractItemView
{
	background-color: rgb(52, 52, 52);
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_addremove">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>431</width>
     <height>491</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_chooseFood">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose Food Item:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_select">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_foodName">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_addDiscountoffer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>371</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_addOffer">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Discount Offer</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_Price">
         <property name="text">
          <string>Price:  Rs.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_showPrice">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>31</y>
        <width>281</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_foodName">
         <property name="text">
          <string>Food Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_showFoodname">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_discoutPrice">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>133</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Discount Price:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_discountPrice">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>130</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>590</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
